game.rb

```diff
@@ -0,0 +1,133 @@
+class Game
+ 
+    attr_accessor :player1 , :player2 , :board_array , :current_player_move
+
+    def initialize(player1,player2)
+        @player1 = player1
+        @player2 = player2
+        @board_array = Array.new(3){Array.new(3,"_")} 
+    end
+
+
+    def move(i)
+        if i%2==0
+            puts "Player #{@player1.name}'s turn: "
```

Comment:
Move this `puts` into the `prompts.rb` class. This is because it violates the [Single Responsibility](https://stackify.com/solid-design-principles/) principle of OOP here.

Reply:
Do this for all other `puts` in this file.